ClinicWeb/src/main/java/pl/infoshare/clinicweb/file/GeneratorData.java

```diff
@@ -0,0 +1,108 @@
+package pl.infoshare.clinicweb.file;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.javafaker.Faker;
+import io.github.viepovsky.polishutils.pesel.Pesel;
+import io.github.viepovsky.polishutils.pesel.PeselGenerator;
+import io.github.viepovsky.polishutils.pesel.PeselGeneratorParams;
+import org.springframework.ui.Model;
+import pl.infoshare.clinicweb.doctor.Doctor;
+import pl.infoshare.clinicweb.doctor.Specialization;
+import pl.infoshare.clinicweb.patient.Address;
+import pl.infoshare.clinicweb.patient.Patient;
+import pl.infoshare.clinicweb.user.Gender;
+import pl.infoshare.clinicweb.user.PersonDetails;
+
+import java.time.LocalDate;
+import java.util.*;
+
+public class GeneratorData {
+    static PeselGeneratorParams.Gender gender = PeselGeneratorParams.Gender.FEMALE;
+
+    private static final int count = 15;
+
+    static String PATIENT_PATH = "ClinicWeb/src/main/resources/patients.json";
+    static String DOCTOR_PATH = "ClinicWeb/src/main/resources/doctors.json";
+
+    static Specialization[] specializations = Specialization.values();
+    static List<Specialization> list = Arrays.stream(specializations).toList();
+
+    static Random rand = new Random();
+    static Faker faker = new Faker(new Locale("pl"));
+
+
+    private static PersonDetails generatePersonDetails() {
+        String firstName = faker.name().firstName();
+        String surname = faker.name().lastName();
+        String phoneNumber = faker.phoneNumber().phoneNumber();
+        String generatedPesel = PeselGenerator.generatePeselStatic();
+        Pesel pesel = new Pesel(generatedPesel);
+        LocalDate birthDate = pesel.getBirthDate();
+        gender = PeselGeneratorParams.Gender.FEMALE;
+        Gender genderName = Gender.WOMAN;
+        return new PersonDetails(firstName, surname, phoneNumber, generatedPesel, birthDate, genderName);
```

Comment:
```suggestion
    private static PersonDetails generatePersonDetails() {
    PersonDetails details = new PersonDetails();
        details.setName(faker.name().firstName());
        details.setSurname(faker.name().lastName());
        ... itd.
        
        return details ;
```
dla wszystkich obiektów